workshop/app/templates/application.hbs

```diff
@@ -1,3 +1,9 @@
-<h2 id="title">Welcome to Ember.js</h2>
+<div class="jumbotron">
+  <div class="container">
+    <h1>Bernice's Blog</h1>
```

Comment:
Some person with a blog. I just generated a name from http://pseudorandom.name/ and it happened to have alliteration.

Should we use a different name or no name?